src/test/e2e/e2e_zarf_create_test.go

```diff
@@ -0,0 +1,22 @@
+package test
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestE2eZarfCreate(t *testing.T) {
+	defer e2e.cleanupAfterTest(t)
+
+	// run `zarf create` with a specified image cache location
+	imageCachePath := "/tmp/.image_cache-location"
```

Comment:
At some point in the test we should add this path to the `e2e.filesToRemove` list so it can get cleaned up from the host.

Reply:
done!